lerobot/__init__.py

```diff
@@ -61,13 +61,21 @@
         "lerobot/aloha_sim_insertion_scripted",
         "lerobot/aloha_sim_transfer_cube_human",
         "lerobot/aloha_sim_transfer_cube_scripted",
+        "lerobot/aloha_sim_insertion_human_image",
```

Comment:
@AshisGhosh noticed that training on these `_image` datasets yielded 0 eval performance. Has this been addressed?

Reply:
@AshisGhosh Left is `_image` dataset, right is video dataset.

<img width="1493" alt="Screenshot 2024-05-19 at 23 29 55" src="https://github.com/huggingface/lerobot/assets/4681518/263b77bb-1ee7-4b35-a8b3-dcafe7fd6270">

Left:
```bash
python lerobot/scripts/train.py \
policy=act \
env=aloha \
env.task=AlohaTransferCube-v0 \
dataset_repo_id=lerobot/aloha_sim_transfer_cube_human_image \
hydra.run.dir=outputs/train/aloha_act_sim_transfer_cube_human_image \
hydra.job.name=aloha_act_sim_transfer_cube_human_image \
wandb.enable=true
```

Right:
```bash
python lerobot/scripts/train.py \
policy=act \
env=aloha \
env.task=AlohaTransferCube-v0 \
dataset_repo_id=lerobot/aloha_sim_transfer_cube_human \
hydra.run.dir=outputs/train/aloha_act_sim_transfer_cube_human \
hydra.job.name=aloha_act_sim_transfer_cube_human \
wandb.enable=true
```